cli/azd/pkg/output/ux/environment_details.go

```diff
@@ -17,15 +17,22 @@ type EnvironmentDetails struct {
 }
 
 func (t *EnvironmentDetails) ToString(currentIndentation string) string {
-	var location string
+	details := []string{}
+	if t.Subscription != "" {
+		details = append(details, fmt.Sprintf(
+			"Subscription: %s",
+			output.WithHighLightFormat(t.Subscription),
+		))
+	}
+
 	if t.Location != "" {
-		location = fmt.Sprintf("\nLocation: %s", color.BlueString(t.Location))
```

Comment:
Normalized to use our Highlight color.

Reply:
Should verify with @Austinauth which color is the right one to use in-context

Reply:
Roll back for now.

Reply:
Just left this as unresolved since it still shows up as `WithHighlightFormat` for me, just making sure the change is intended.